### PR TITLE
config: enable tichi for rfcs

### PR DESCRIPTION
**Pull request by hi-rustin**

### CODE REVIEW

**Comment by hi-rustin** [comment]
/assign @Mini256 

PTAL~ Thanks!

File: prow/config/plugins.yaml [hunk 1]
```diff
@@ -219,7 +219,27 @@ external_plugins:
...
+        - pull_request
+    - name: ti-community-merge
+      events:
+        - issue_comment
+        - pull_request_review_comment
+        - pull_request
+    - name: ti-community-autoresponder
+      events:
+        - issue_comment
+        - pull_request_review_comment
```
Comment (Mini256) [inline prow/config/plugins.yaml:237]: Is there any place to use this plugin?

**Review by Mini256 — COMMENTED** [review]
And should we add plugin such as `assign`?

File: prow/config/plugins.yaml (hunk 1 above)
Comment (hi-rustin) [reply prow/config/plugins.yaml:237]: Removed.

**Comment by hi-rustin** [comment]
> And should we add plugin such as `assign`?

Added.

**Review by Mini256 — APPROVED** [review]

**Pull request merged**